packages/webcert/src/store/srs/srsReducer.tsx

```diff
@@ -10,35 +10,56 @@ import {
   updatePatientId,
   updateSrsInfo,
   updateSickLeaveChoice,
+  updateSrsQuestions,
+  updateSrsPredictions,
+  updateRiskOpinion,
+  updateUnitId,
+  updateCareProviderId,
```

Comment:
I get error saying this action is not exported from './srsActions' when trying to build

Reply:
Now it should work